scripts/abins/sdata.py

```diff
@@ -152,6 +183,32 @@ def get_empty(cls: SD, *,
     def get_frequencies(self) -> np.ndarray:
         return self._frequencies.copy()
 
+    def get_q_bins(self) -> np.ndarray:
+        if self._q_bins is None:
+            return None
+        else:
+            return self._q_bins.copy()
+
+    def get_q_bin_centres(self) -> np.ndarray:
+        q_bins = self.get_q_bins()
+        if q_bins is None:
+            raise ValueError("No q_bins on this SData")
+        else:
+            return (q_bins[1:] + q_bins[:-1]) / 2.
+
+    def set_q_bins(self, q_bins: np.ndarray) -> None:
+        """Update q-bins stored on SData
+
+        Args:
+            q_bins: 1-D set of q-point bin edges surrounding rows in S data
+                (e.g. for q-bins=[1., 2., 3.], S arrays will have two rows
+                 corresponding to q=1-2 Å^-1, q=2-3 Å^-1; typically these
```

Comment:
```suggestion
                 corresponding to q =1 to 2 Å^-1, q =2 to 3 Å^-1; typically these
```
just so no-one tries to read it as 1 minus 2